tutorials/ui/gui_skinning.rst

```diff
@@ -48,32 +23,125 @@ user interfaces.
    Don't use the custom theme option in the Project Settings, as there
    are known bugs with theme propagation. Instead, apply your theme to the
    root Control node's Theme property. It will propagate to instanced scenes
-   automatically. To get correct theming in the editor for instanced scenes,
+   automatically. To get correct themeing in the editor for instanced scenes,
```

Comment:
```suggestion
   automatically. To get correct theming in the editor for instanced scenes,
```

I think this spelling is actually correct (or at least more widely used), i.e. https://en.wikipedia.org/wiki/Theming